lang/python/airr/interface.py

```diff
@@ -6,10 +6,14 @@
 import pandas as pd
 from collections import OrderedDict
 from itertools import chain
+from pkg_resources import resource_filename
+import json
+import yaml
+import yamlordereddictloader
```

Comment:
I can't recall why right now, but I had issues with `yamlordereddictloader` in recent PyYAML versions (needed to solve a warning).  What I did instead was:

```
# Set YAML loader to OrderedDict
from collections import OrderedDict
def dict_representer(dumper, data):  return dumper.represent_dict(data.iteritems())
def dict_constructor(loader, node):  return OrderedDict(loader.construct_pairs(node))
yaml.add_representer(OrderedDict, dict_representer)
yaml.add_constructor(yaml.resolver.BaseResolver.DEFAULT_MAPPING_TAG, dict_constructor)
```

Then  you can load the yaml with the FullLoader and it'll be ordered:

```
yaml.load(handle, Loader=yaml.FullLoader)
```

Reply:
I copied the code from `schema.py`. I'm using PyYaml 5.1.2 and it seems to be working just fine?

Reply:
Okay, then let's not worry about it? If the problem (you know, that one I can't remember) pops up we can deal with it then.